utilities/Make JSONs.ipynb

```diff
@@ -48,10 +48,10 @@
        "      <th>PDF Resume</th>\n",
```

Comment:
May be good to clear the output for the notebook.

Reply:
Good call here